manifests/04-rbac-rolebinding.yaml

```diff
@@ -65,3 +65,16 @@ subjects:
   - kind: Group
     name: system:authenticated
     apiGroup: rbac.authorization.k8s.io
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-extensions
+roleRef:
+  kind: Role
+  name: console-extensions
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: Group
+  name: system:authenticated
```

Comment:
This is clearly the wrong way to implement this.

Reply:
@bparees